docs/configuration/entrypoints.md

```diff
@@ -34,6 +34,9 @@ To redirect an http entrypoint to an https entrypoint (with SNI support).
       KeyFile = "integration/fixtures/https/snitest.org.key"
 ```
 
+!!! note
+    `regex` and `replacement` have not to be set in `redirect` structure during entrypoint redirection (they are not used).
```

Comment:
I would use the wording:

Please note that `regex` and `replacement` do not have to be set in the redirect structure if an entrypoint is defined for the redirection (they will not be used in this case)